src/ResourceManager/RecoveryServices/Commands.RecoveryServices/Vault/SetAzureRmRecoveryServicesBackupProperties.cs

```diff
@@ -48,11 +48,11 @@ public override void ExecuteCmdlet()
         {
             try
             {
-                if (!string.IsNullOrEmpty(this.BackupStorageRedundancy))
+                if (this.BackupStorageRedundancy.HasValue)
```

Comment:
if BackupStorageRedundancy is not set by customer, then this cmdlet exits as success with no-op. Do you want to throw any exception saying storageRedundancy input is must (in future - one of redundancy or dedupe is must)